### PR TITLE
HELP:Update json from code.

### PR DESCRIPTION
**Pull request by hemostx**
Ran dev_help_issues generate for an updated base to work on help strings.
- A bunch of vars/properties got automatically reordered alphabetically.
- A lot of new items to be described have now been system-generated.

### CODE REVIEW

File: help_macros.json [hunk 1]
```diff
@@ -304,25 +304,25 @@
...
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
   },
   "powerups": {
     "flags": [
       "incomplete"
     ],
-    "teamplay-restricted": true
+    "teamplay-restricted": false
```
Comment (meag) [inline help_macros.json:325]: Hmmm... this is definitely a bug, but what ruleset did you have enabled when you generated the help file? :)

**Comment by meag** [comment]
Think you've just highlighted a bug :)  Teamplay restrictions are disabled on these macros, but that's just the current state depending on ruleset?

**Comment by hemostx** [comment]
Aha! that was in the default config - default ruleset.
I figured the `generate` command picked from whatever the default is set to, as is suggested by a few vars I checked when I saw what was being removed.

**Comment by hemostx** [comment]
This property does not show up when I `/describe $macro` - only the description does. Maybe that command should actually output a little more for macros?
Hold that thought - this patch showed that /describe does not pull the default values from json but from what the default is currently registered as in memory. The values in there are only useful to an external tool using the json (ie the forthcoming autogenerated reference pages)... so this is where these details will surface for now.

**Comment by meag** [comment]
Pulled in, had to manually fix conflicting files

**Pull request closed without merging**